lib/client.js

```diff
@@ -298,7 +298,7 @@ Client.prototype._attachApi = function () {
  *  @memberof Client
  *  @method start
  */
-Client.prototype.start = function (apps, callback) {
+Client.prototype.start = function (apps, subscribeAll, callback) {
```

Comment:
Probably need to still support the old method signature.

    if (typeof subscribeAll === 'function') {
        callback = subscribeAll;
        subscribeAll = null;
    }